nixos/tests/home-assistant.nix

```diff
@@ -14,9 +14,10 @@ in {
     environment.systemPackages = with pkgs; [ mosquitto ];
     services.mosquitto = {
       enable = true;
+      checkPasswords = true;
```

Comment:
This is required so the nixos module includes the password file, that is rendered anyway. :facepalm: